app/src/pages/inside/dashboardItemPage/widgetWizardModal/widgetControls/controls/filtersControl/filtersItem/filtersItem.jsx

```diff
@@ -41,13 +43,20 @@ export class FiltersItem extends PureComponent {
           onChange={onChange}
           circleAtTop
         >
-          <FilterName search={search} filter={filter} userId={userId} showDesc={false} />
+          <FilterName
+            search={search}
+            filter={filter}
+            userId={userId}
+            showDesc={false}
+            editable={editable}
+          />
           <FilterOptions entities={filter.conditions} sort={filter.orders}>
-            {userId === filter.owner && (
-              <span className={cx('pencil-icon')} onClick={onEdit}>
-                {Parser(PencilIcon)}
-              </span>
-            )}
+            {userId === filter.owner &&
+              editable && (
```

Comment:
I don't see any place where this flag is passed to the component. Are all filters unavailable for edit in this modal now?
Looks like a mistake.

Reply:
I have checked already and i have seen that filtersItem only called as part of FiltersControl. The FiltersControl called only from the widgetControl components, so based on the usage (we dont need the edit icon) i set the "editable" option visibility false as default. I think it is more logical, cause an item can be occurred at many place, but in an editable form it should be occurred only one place. So to avoid more code (give editable value as an extra parameter) i prefer my solution as default the "filtersItem" is not editable.

Reply:
>  so based on the usage (we dont need the edit icon)
If current user is the owner of this filter, this filter should be editable.

The bug is that we have 2 "edit" icons: one in the name row (`filterName.jsx`), and one in the options row (in this file). Your change in the `filterName.jsx` is enough to hide the unneeded icon. Don't hide the second one - it works correctly.

Reply:
OK, seems that part wasn't clear for me, thanks, i will correct it.